rust/osm2lanes/src/tag/mod.rs

```diff
@@ -53,6 +50,50 @@ impl Tags {
             .collect::<Vec<String>>()
     }
 
+    // TODO, shouldn't TagKey be passed by reference?
```

Comment:
I feel like I tried to do this conversion a few weeks back and failed, but theoretically `AsRef<TagKey>` could work

Reply:
I managed: https://github.com/droogmic/osm2lanes/pull/1/files